testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java

```diff
@@ -91,6 +91,14 @@ public static String getTableCommentOnDelta(String schemaName, String tableName)
                 .collect(onlyElement());
     }
 
+    public static String getTablePropertyOnDelta(String schemaName, String tableName, String key)
```

Comment:
This commit is not anymore necessary - see https://github.com/trinodb/trino/blob/b91112a2004997c250b88b4f564dbea0175eca81/testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java#L95-L99

Pls rebase on `master`.